### PR TITLE
chore(release): 3.0.0

### PR DESCRIPTION
**Pull request by ssvegaraju**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fhir-works-on-aws-authz-smart",
-  "version": "2.2.3",
+  "version": "2.2.4",
```
Comment (rsmayda) [inline package.json]: Looks like Yanyu is about to merge a change too, could we wait and include hers?

https://github.com/awslabs/fhir-works-on-aws-authz-smart/pull/60

Comment (ssvegaraju) [reply package.json]: yup! I'll sync with Yanyu to coordinate this.

Comment (ssvegaraju) [reply package.json]: Not sure why the version jumped to 3.0.0... is this expected? If not I will delete this branch and restart the release from scratch.

Comment (rsmayda) [reply package.json]: Lets sync with Yanyu on this -- it looks like her change is a breaking one

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+### Features
+
+* update system export to only allow system scope by default ([#60](https://github.com/awslabs/fhir-works-on-aws-authz-smart/issues/60)) ([45b7960](https://github.com/awslabs/fhir-works-on-aws-authz-smart/commit/45b796031c36960b8f5be1b765beee839d0d9aa8))
+
+
+### Bug Fixes
+
+* remove filtered responses from total count ([#63](https://github.com/awslabs/fhir-works-on-aws-authz-smart/issues/63)) ([82b6ff3](https://github.com/awslabs/fhir-works-on-aws-authz-smart/commit/82b6ff372b44ca40dc3575e7505695f79747d6de))
+
+### [2.2.4](https://github.com/awslabs/fhir-works-on-aws-authz-smart/compare/v2.2.3...v2.2.4) (2021-09-13)
```
Comment (Bingjiling) [inline CHANGELOG.md]: Do we still need this version?

Comment (ssvegaraju) [reply CHANGELOG.md]: I can remove this since it also comes under the release notes for 3.0.0 👍

**Review by rsmayda — COMMENTED** [review]
LGTM @Bingjiling Please make sure this looks right too

**Review by Bingjiling — APPROVED** [review]

**Pull request merged**